napari/_qt/_tests/test_qt_viewer.py

```diff
@@ -334,7 +334,15 @@ def test_export_rois(make_napari_viewer, tmp_path):
         (tmp_path / f'roi_{i}.png').exists()
         for i in range(len(roi_shapes_data))
     )
-    assert all(roi.shape == (20, 20, 4) for roi in test_roi)
+
+    # we get scaling to adjust the expected size of ROI images
+    # and number of white pixels in the ROI screenshots
+    # it may not work for screens with fractional scaling
+    scaling = viewer.window._qt_window.screen().devicePixelRatio()
```

Comment:
@willingc Could you check this comment? I especially would like to point that this fix may not work for screens with fractional scaling.

Reply:
@Czaki left a suggestion. 👍🏼